distributed/dashboard/components/shared.py

```diff
@@ -532,12 +569,22 @@ def __init__(self, worker, height=150, **kwargs):
         self.worker.monitor.update()
 
     def get_data(self):
-        d = self.worker.monitor.range_query(start=self.last)
+        d = self.worker.monitor.range_query(start=self.last_count)
         d["time"] = [x * 1000 for x in d["time"]]
-        self.last = self.worker.monitor.count
+        self.last_count = self.worker.monitor.count
         return d
 
     @without_property_validation
     def update(self):
         with log_errors():
             self.source.stream(self.get_data(), 1000)
+            self.label_source.data["cpu"] = list(
+                "{}: {:.1f}%".format(f.__name__, f(self.source.data["cpu"]))
+                for f in [min, max, mean]
+            )
+            self.label_source.data["memory"] = list(
+                "{}: {}".format(
+                    f.__name__, dask.utils.format_bytes(f(self.source.data["memory"]))
```

Comment:
Using `format_bytes` here results in slightly different output of the human readable memory from the y-ticks - is this okay?

Reply:
What are the differences ?

Reply:
`123.4MiB` (y-ticks) versus `123.4 MB` (`format_bytes`)

Reply:
I don't have a strong opinion here.  What do you think ?

Reply:
My preference is `format_bytes`, although I'm sure if this is a problem later on it shouldn't be hard to find an alternative formatter.